scripts/install.js

```diff
@@ -21,58 +21,51 @@ const rimraf = require('rimraf');
 const tar = require('tar');
 const util = require('util');
 const zip = require('adm-zip');
+const cp = require('child_process');
+const os = require('os');
+const ProgressBar = require('progress');
+const {depsPath, depsLibPath} = require('./deps-constants.js');
 
-const copy = util.promisify(fs.copyFile);
 const exists = util.promisify(fs.exists);
 const mkdir = util.promisify(fs.mkdir);
-const rename = util.promisify(fs.rename);
 const rimrafPromise = util.promisify(rimraf);
-const symlink = util.promisify(fs.symlink);
 const unlink = util.promisify(fs.unlink);
+const exec = util.promisify(cp.exec);
 
 const BASE_URI = 'https://storage.googleapis.com/tf-builds/';
 const CPU_DARWIN = 'libtensorflow_r1_10_darwin.tar.gz';
 const CPU_LINUX = 'libtensorflow_r1_10_linux_cpu.tar.gz';
 const GPU_LINUX = 'libtensorflow_r1_10_linux_gpu.tar.gz';
 const CPU_WINDOWS = 'libtensorflow_r1_10_windows_cpu.zip';
 
-const platform = process.argv[2];
-let action = process.argv[3];
```

Comment:
This file has formatting errors - are you using VSCode with format-on-save + clang?

Reply:
I'm using visual studio on my windows and this is the reason for format. I'll install VSCode and use it to fix format.